migration/migrating-3-4/migrating-openshift-3-to-4.adoc

```diff
@@ -107,14 +99,13 @@ include::modules/migration-understanding-cpma.adoc[leveloffset=+2]
 include::modules/migration-installing-cpma.adoc[leveloffset=+2]
 include::modules/migration-using-cpma.adoc[leveloffset=+2]
 
-== Troubleshooting a failed migration
+== Troubleshooting
 
 You can view the migration custom resources (CRs) and download logs to troubleshoot a failed migration.
```

Comment:
s/custom resources (CRs)/CRs/
If you spell out on first instance at the beginning of assembly (line 11), you should use just the abbreviation here.